docs/installguide/install_all.rst

```diff
@@ -44,6 +44,10 @@ For more advanced use of KA Lite, such as changing the default port, see :ref:`r
 or use the command-line ``kalite`` program, which in typical installations can be found at the path
 ``C:\Python27\Scripts\kalite``. Run ``kalite --help`` for usage info.
 
+.. warning::
+    Using existing KA Lite application and download non English language with another ``computer account`` will cause improper loading of exercise images.

```

Comment:
My suggestion:

```
If you need to download and install contentpacks locally for languages other than English, make sure you are doing it **as the same user that installed KA Lite** in the first place. If you perform the contentpack installation as a different user, some content will not load properly.
```
I would also recommend that `install contentpacks locally` be linked with the relevant item in the FAQ:

http://ka-lite.readthedocs.io/en/0.16.x/faq.html#how-can-i-install-a-language-pack-without-a-reliable-internet-connection